x11-misc/trayer-srg/trayer-srg-1.1.8-r1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Gentoo Authors
```

Comment:
Please also keep the old copyright header since the ebuild hasn't changed that much.